pandokia/helpers/runner_unit2.py

```diff
@@ -20,6 +21,17 @@
 
 from pandokia.helpers import pycode
 
+# replacement for imp.load_source() from https://docs.python.org/dev/whatsnew/3.12.html#imp
```

Comment:
how many places WITHIN pandokia's tree?

Reply:
There were three, now there is only one.